AssetTracking/AssetTracking/Helpers/GraphService.cs

```diff
@@ -0,0 +1,189 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Graph;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace AssetTracking.Helpers
+{
+    public static class GraphService
+    {
+        // Loads the user's profile in formatted JSON.
+        public static async Task<string> GetUserJson(GraphServiceClient graphClient, string email, HttpContext httpContext)
+        {
+            if (email == null) return JsonConvert.SerializeObject(new { Message = "Email address cannot be null." }, Formatting.Indented);
```

Comment:
For ease of code readability and/or maintenance, have the `return` code in its own line or enclosed in double braces.